### PR TITLE
removed unused DITTO_POLICIES_ENFORCER_CACHE_MAX_SIZE in thingssearch

### PR DESCRIPTION
**Pull request by kalinkostashki**
- this property is only used for the policies service in CachingPolicyEnforcerProvider.java and PolicyEnforcerProviderExtension.java

### CODE REVIEW

File: deployment/helm/ditto/templates/thingssearch-deployment.yaml [hunk 1]
```diff
@@ -254,8 +254,6 @@ spec:
               value: "{{ .Values.thingsSearch.config.updater.backgroundSync.throttle.throughput }}"
             - name: BACKGROUND_SYCN_THROTTLE_PERIOD
               value: "{{ .Values.thingsSearch.config.updater.backgroundSync.throttle.period }}"
-            - name: DITTO_POLICIES_ENFORCER_CACHE_MAX_SIZE
-              value: "{{ .Values.thingsSearch.config.updater.stream.policiesEnforcer.cache.maxSize }}"
             - name: DITTO_POLICIES_ENFORCER_CACHE_EXPIRE_AFTER_WRITE
```
Comment (thjaeckle) [inline deployment/helm/ditto/templates/thingssearch-deployment.yaml:257]: @kalinkostashki shouldn't those keys then also be deleted?

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Thanks @kalinkostashki for the PR - you're right - those are not used in things-search.

However, there are also more config keys to the `DITTO_POLICIES_ENFORCER_CACHE_*`- we should also remove them, right?

And what about the Helm `values.yaml`? 
`.Values.thingsSearch.config.updater.stream.policiesEnforcer.cache.` can likely be removed from that as well, right?

File: deployment/helm/ditto/templates/thingssearch-deployment.yaml (hunk 1 above)
Comment (kalinkostashki) [reply deployment/helm/ditto/templates/thingssearch-deployment.yaml:257]: yes, you're right I forgot about them.
will remove them as well :)

**Comment by kalinkostashki** [comment]
> Thanks @kalinkostashki for the PR - you're right - those are not used in things-search.
> 
> However, there are also more config keys to the `DITTO_POLICIES_ENFORCER_CACHE_*`- we should also remove them, right?
> 
> And what about the Helm `values.yaml`? `.Values.thingsSearch.config.updater.stream.policiesEnforcer.cache.` can likely be removed from that as well, right?

I don't we should remove Values.thingsSearch.config.updater.stream.policiesEnforcer.cache. as those are used to set THINGS_SEARCH_UPDATER_STREAM_POLICY_CACHE* options.

**Comment by thjaeckle** [comment]
> I don't we should remove Values.thingsSearch.config.updater.stream.policiesEnforcer.cache. as those are used to set THINGS_SEARCH_UPDATER_STREAM_POLICY_CACHE* options.

Oh, good catch - then they were used by 2 places .. which is already a bad smell :D

Your changes LGTM 👍

**Review by thjaeckle — APPROVED** [review]

**Pull request merged**